### ISSUE TITLE
Importing submodels in pandas/core/series.py result with an error

### ISSUE THREAD

**Issue by hechtlinger** [pandas-dev/pandas#21319]
#### Code Sample, a copy-pastable example if possible

```python
import pandas.core.ops as ops
import pandas.core.algorithms as algorithms

import pandas.core.common as com
import pandas.core.nanops as nanops
import pandas.io.formats.format as fmt

```
#### Problem description

The above code in series.py resulted with an import error. It can simply be replaced with:
```python
from pandas.core import ops, algorithms, com, nanops, fmt
```
Or however aesthetically pleasing.

**Comment by jreback** [comment]
what exactly resulting in a error? what environment??

**Comment by jreback** [comment]
what are you doing when u get an error? what version?

**Comment by hechtlinger** [comment]
I installed a new version on python 3.6 using anaconda. Right after I "pip install tensorflow". Tried to import tensorflow and got this import error. I created a one line python script calling the command
```python
import pandas.core.ops as ops
```
to test it and it failed. After changing the import command to
```python
from pandas.core import ops
```
it seems to works fine. I can close this issue if it's unrelated bug on my python installation. It works now, and It does work properly on the other python 2.7 I tested it on.
